src/databricks/labs/lsql/dashboards.py

```diff
@@ -897,32 +897,32 @@ def _from_dashboard_folder(cls, folder: Path) -> "DashboardMetadata":
             tiles.append(tile)
         return cls(display_name=folder.name, _tiles=tiles)
 
-    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path) -> Path:
+    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path, zip_file_name: str) -> Path:
         """Export the dashboard queries to CSV files directly into a ZIP archive."""
-        zip_export = export_path / "export_to_zipped_csv.zip"
+        zip_export = export_path / f"{zip_file_name}.zip"
 
         with ZipFile(zip_export, mode="w") as zip_file:
             for tile in self.tiles:
-                if tile.metadata.is_query():
-                    rows = sql_backend.fetch(tile.content)
+                if not tile.metadata.is_query():
+                    continue
 
```

Comment:
Nit: remove a couple of the whitelines of code blocks logically belonging to gether
```suggestion
```